zerver/webhooks/runscope/tests.py

```diff
@@ -0,0 +1,65 @@
+from zerver.lib.test_classes import WebhookTestCase
+
+class RunscopeHookTests(WebhookTestCase):
+    STREAM_NAME = 'Runscope'
```

Comment:
We prefer using lowercase stream names so it would be preferable to change this to `runscope`.